API/BDD/features/step-definitions/steps.js

```diff
@@ -0,0 +1,92 @@
+const { Given, Then } = require('@wdio/cucumber-framework');
+const { expect } = require('@wdio/globals')
+const axios = require('axios');
+
+let response;
+
+Given(/^Send request to "([^"]*)"$/, async (url) => {
+	const user = {
+        "id": 22,
+        "username": "Test UserLastName",
+        "firstName": "Test",
+        "lastName": "UserLastName",
+        "email": "testuser@example.com",
+        "password": "test12356_)",
+        "phone": "+375297891234",
+        "userStatus": 0
```

Comment:
Make object once and use when need it multiple times

Reply:
Declared the object at the beginning